flink-streaming-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/proxy/KinesisProxyInterface.java

```diff
@@ -34,14 +34,16 @@
 	 *
 	 * @param shard the shard to get the iterator
 	 * @param shardIteratorType the iterator type, defining how the shard is to be iterated
-	 *                          (one of: TRIM_HORIZON, LATEST, AT_SEQUENCE_NUMBER, AFTER_SEQUENCE_NUMBER)
-	 * @param startingSeqNum sequence number, must be null if shardIteratorType is TRIM_HORIZON or LATEST
+	 *                          (one of: TRIM_HORIZON, LATEST, AT_TIMESTAMP, AT_SEQUENCE_NUMBER, AFTER_SEQUENCE_NUMBER)
+	 * @param startingMarker is null if shardIteratorType is TRIM_HORIZON or LATEST,
+	 *                       is as a timestamp if shardIteratorType is AT_TIMESTAMP,
+	 *                       is as a sequence number if shardIteratorType is AT_SEQUENCE_NUMBER, AFTER_SEQUENCE_NUMBER
```

Comment:
I would suggest the following changes to the Javadoc here to be more specific:

is null --> "should be {@code null} if ..."
is as a timestamp --> "should be a {@code Date} value if ..."
is as a sequence number --> "should be a {@code String} representing the sequence number if ..."